FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/BaseActivity.kt

```diff
@@ -126,6 +127,19 @@ abstract class BaseActivity<T : ViewBinding> : AppCompatActivity() {
 
   }
 
+  /**
+   * This method can bu used to handle destination changes.
+   *
+   * @param controller the controller that navigated
+   * @param destination the new destination
+   * @param arguments the arguments passed to the destination
+   */
+  protected open fun onDestinationChanged(
+    controller: NavController, destination: NavDestination, arguments: Bundle?
+  ) {
+
```

Comment:
Please add comment like `// nothing to do here, actual actions will be defined by subclasses` if there is really nothing to do here.